tools/toc.sh

```diff
@@ -9,7 +9,7 @@
 let ERROR=0
 
 # Find all markdown files
-for f in $(find . | grep .md); do
+for f in $(find doc/ boards/ kernel/ arch/ capsules/ chips/ userland/ -name "*.md"); do
```

Comment:
Oh wait... yes... it's still enumerating all of the files and piping it through grep. `find * -name "*.md"` does the trick though